src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -106,24 +103,19 @@ export class ResultsViewPageStore {
         }
     }, []);
 
-    readonly mskImpactGermlineConsentedPatientIds = remoteData({
-        await: () => [this.patientIds],
+    readonly germlineConsentedSampleIds = remoteData({
         invoke: async () => {
-            const clinicalDataSingleStudyFilter = {
-                attributeIds: [IMPACT_GERMLINE_TESTING_CONSENT],
-                ids: this.patientIds.result
-            };
-            const clinicalDataResponse = await fetchClinicalDataInStudy(
-                this.studyId, clinicalDataSingleStudyFilter, 'PATIENT'
-            );
-            if (clinicalDataResponse) {
-                return _.uniq(clinicalDataResponse.map(
-                    (cd:ClinicalData) => cd.entityId)
-                );
+            if (this.germlineSampleListId) {
+                return await client.getAllSampleIdsInSampleListUsingGET({
+                    sampleListId: this.germlineSampleListId
+                });
             } else {
                 return [];
             }
         },
+        onError: () => {
+            // fail silently
```

Comment:
Since for most of the studies we don't have `${this.studyId}_germline` sample list id, most of the times we end up with an API error, and ignore it. I don't think this is the best solution...